test/e2e/nodelatencymonitor_test.go

```diff
@@ -0,0 +1,194 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	v1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	statsv1alpha1 "antrea.io/antrea/pkg/apis/stats/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	nodeLatencyMonitorName = "default"
+	nodeLatencyMonitorNS   = "kube-system"
+	pingIntervalSeconds    = 50
+)
+
+func TestNodeLatencyMonitor(t *testing.T) {
+	skipIfNotRequired(t)
+
+	data, err := setupTest(t)
+	if err != nil {
+		t.Fatalf("Failed to set up clients: %v", err)
+	}
+	defer teardownTest(t, data)
+
+	createOrUpdateNodeLatencyMonitorCR(t, data)
+
+	t.Run("testEnableLatencyProbes", func(t *testing.T) {
+		testEnableLatencyProbes(t, data)
+	})
+	t.Run("testRetrieveLatencyStats", func(t *testing.T) {
+		testRetrieveLatencyStats(t, data)
+	})
+	t.Run("testUpdatePingInterval", func(t *testing.T) {
+		testUpdatePingInterval(t, data)
+	})
+	t.Run("testDisableLatencyProbes", func(t *testing.T) {
+		testDisableLatencyProbes(t, data)
+	})
```

Comment:
it's better if subtests are independent and don't assume a specific execution order